hydra_agent/sync_mechanism.py

```diff
@@ -0,0 +1,153 @@
+import urllib.request
+import json
+import logging
+from urllib.error import URLError, HTTPError
+from redis_proxy import RedisProxy
+from graphutils import GraphUtils
+from redisgraph import Graph, Node
+from requests import Session
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
`basicConfig` has to be declared once at the top of the package, then every module/package can declare its own `getLogger`

Reply:
I removed basicConfig and I'm using as we're on hydrus on top the module:

`logger = logging.getLogger(__file__)`

Is it okay like that?

Reply:
For now is ok, we will fix all the client configuration when addressing the proper issue